src/styles/sidebar/components/markdown-editor.scss

```diff
@@ -0,0 +1,44 @@
+$toolbar-border: 0.1em solid $grey-3;
+
+.markdown-editor__toolbar {
+  display: flex;
+  flex-direction: row;
+
+  background-color: $white;
+  border-top: $toolbar-border;
+  border-left: $toolbar-border;
+  border-right: $toolbar-border;
+  border-radius: 0.15em 0.15em 0 0;
+  width: 100%;
+  margin-bottom: -0.1em;
+  padding: 5px 5px;
+  user-select: none;
+}
+
+.markdown-editor__toolbar-button {
+  appearance: none;
+  border: none;
+  background: none;
+
+  color: $grey-5;
+  font-size: 16px;
+
+  &:hover,
+  &:focus {
+    color: black;
```

Comment:
We're ok with `black` (and/or `white`) without variables?

Also, I didn't know `black` was valid to use in the UI! TIL.

Reply:
We did have this debate in the past. I advocated for variables for all colors at the time, but now I think about it this is a bit pointless (using `$white` instead of just `white`).